CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ if(DOWNLOAD_HIREDIS)
   include(FetchContent)
   FetchContent_Declare(hiredis
     GIT_REPOSITORY https://github.com/redis/hiredis
-    GIT_TAG        v1.0.0
+    GIT_TAG        f8de9a4bd433791890572f7b9147e685653ddef9
```

Comment:
So we require an unreleased hiredis version? That's worth mentioning in the PR description.

Reply:
Good point. I might have found an alternative solution. hiredis is lacking some targets before the https://github.com/redis/hiredis/pull/1005, but it might be good to handle the released versions so will add a check and fix.

Reply:
Now handled by adding the missing targets in hiredis-clusters CMakeLists.txt, reverting the used version changes.